oat-sa/extension-tao-itemqti
# feat: include rich passage stylesheet to compiled delivery data

How to test:
- Create or import a test containing styled passages.
- Publish a local delivery
- Login as a test taker 
- Check that styles are applied

The functionality was tested on CG test runner, we still need to deploy a new integration environment to test remote publishing and NextGen test runner.
